src/app/containers/MostRead/Canonical/LastUpdated.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { latin } from '@bbc/gel-foundations/scripts';
+import LastUpdated from './LastUpdated';
+
+let container;
+
+describe('MostReadCanonical - LastUpdated', () => {
+  beforeEach(() => {
+    container = document.createElement('div');
+    document.body.appendChild(container);
+  });
+
+  afterEach(() => {
+    container = null;
+  });
+
+  it(`test LastUpdated renders when timestamp > 60 days`, () => {
```

Comment:
```suggestion
  it(`LastUpdated renders when timestamp > 60 days`, () => {
```

Reply:
Reworded to make it more consistent with other tests that we have:
https://github.com/bbc/simorgh/blob/afa5969a3f3202a4115b05b9433037e2d25bbfbc/src/app/containers/MostRead/Canonical/LastUpdated.test.jsx#L35